### PR TITLE
adding deployment guide to DO managed k8s

### PR DESCRIPTION
**Pull request by jasonmccallister**
### Description
Adds deployment guide for Digital Ocean managed Kubernetes

### Affected components

- [ ] Docs
- [ ] Community Content

### CODE REVIEW

**Comment by jasonmccallister** [comment]
I have also added a bunch of scripts here: https://github.com/jasonmccallister/hasura-digitalocean-kubernetes but did not want to link them in the docs

File: docs/graphql/manual/guides/deployment/digital-ocean-managed-kubernetes.rst [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+.. note::
+
+  Feel free to rename this to match your project name but the remainder of this tutorial will refer to
+  the cluster as ``hasura-demo`` so don't forget to change the name to match!
+
+The following command will create a cluster using the ``doctl`` command and spin up three worker nodes.
+
+.. code-block:: bash
+
+  doctl kubernetes cluster create hasura-demo
```
Comment (marionschleifer) [inline docs/graphql/manual/guides/deployment/digital-ocean-managed-kubernetes.rst]: @jasonmccallister I'm a bit stuck on this command. I get the following error:

```
unable to initialize DigitalOcean api client: access token is required. (hint: run 'doctl auth init')
```

Can we add something here what is the best way to initialise auth?

**Comment by marionschleifer** [comment]
@jasonmccallister just wanted to check with you if you'll have the time to work on this PR 🙂

**Comment by Stefmore02** [comment]
Closing this PR due to merge conflicts. If you consider it should be re-opened, please feel free to do so.

**Pull request closed without merging**